apps/nextra/pages/en/build/sdks/ts-sdk/fetch-data-via-sdk.mdx

```diff
@@ -22,13 +22,16 @@ Many queries have a parameter named `options` to customize the results, use it t
 The `Aptos` client can out of the box query both network data from [fullnodes](https://api.mainnet.aptoslabs.com/v1/spec#/) and the [Indexer](https://cloud.hasura.io/public/graphiql?endpoint=https://api.mainnet.aptoslabs.com/v1/graphql) API which contains aggregated and enriched data. If you want to use a custom query for Indexer API data, you can use `aptos.queryIndexer` like so:
 
 ```ts filename="fetch-data.ts"
-const ledgerInfo = aptos.queryIndexer({
- query: `query MyQuery {
-     ledger_infos {
-       chain_id
-     }
-   }`;
-})
+  const ntf_collection_info = aptos.queryIndexer({
+    query: {
+      query: `query GetLedgerInfo {
+        ledger_infos {
+          chain_id
+        }
+      }`,
+      variables: {}, // If the query does not require variables, it can be an empty object here.
```

Comment:
If the query does not require variables, we can simply not add a `variables` field

https://github.com/aptos-labs/aptos-ts-sdk/blob/main/tests/e2e/api/general.test.ts#L224